HISTORY.rst

```diff
@@ -6,12 +6,19 @@ latest
 ------
 
 Minor changes:
-
+~~~~~~~~~~~~~
 * Added the flag ``--mca btl_vader_single_copy_mechanism none to mpirun in fv3run`` to mpirun in fv3run
 * Add ReadTheDocs configuration file
 * Do not require output dir and fv3config to be remote in ``run_kubernetes``
 * Fix bug when submitting k8s jobs with images that have an "_" in them
 
+Breaking changes
+~~~~~~~~~~~~~~~~
+* Refactored run_kuberentes and run_docker to call run_native via serializing
+  their args/kwargs as json strings. While no API changes occured, the
```

Comment:
API changes did occur, even if the run_native api change was backwards-compatible. A new API was introduced for run_native, and run_kubernetes and run_docker were modified to rely on the new API. Could add "via a new API serializing" above and remove "While no API changes occurred" here.